promql/parser/generated_parser.y

```diff
@@ -376,7 +376,12 @@ paren_expr      : LEFT_PAREN expr RIGHT_PAREN
 
 offset_expr: expr OFFSET duration
                         {
-                        yylex.(*parser).addOffset($1, $3)
+                        yylex.(*parser).addOffset($1, $3, 1)
+                        $$ = $1
+                        }
+                | expr OFFSET SUB duration
```

Comment:
I can rebase and rebuild, but there will always be differences in the goyacc output due to these additional five lines, mainly adding this or | operator and this section to handle the 'OFFSET SUB duration' in opposed to just 'OFFSET duration' section.

Reply:
Yes the difference expected, the conflict is mostly due to the commit history having more changes in the master that you dont have in your branch.